### DISCUSSION TITLE
ES6 Modules Support

### DISCUSSION THREAD

**Discussion by adam3278** [GrapesJS/grapesjs#5870] [Q&A]
Hi,
I saw the .mjs file in dist, so there is the question does the GrapesJS support the ES6 module syntax? Can I load it and use as described for Node environments?

**Comment by artf** [comment]
Hey @adam3278 can you clarify what kind of issue you're facing?

**Comment by adam3278** [comment]
I don’t face any issues, but I’m wondering is grapesjs compatible with es6 modules syntax (import/export) or commonjs only?

**Reply by artf** [reply]
it should be compatible with [both](https://github.com/GrapesJS/grapesjs/blob/53c4b9ee3a00b31b0ba2e8f5224f30f33815c544/package.json#L11-L16)
